docs/reference/api/i18n.md

```diff
@@ -0,0 +1,124 @@
+# i18n REST API
+
+There are two i18n API endpoints to facilitate content management in a multilingual ApostropheCMS project.
```

Comment:
I know this is reference, but there should be a paragraph clarifying that these APIs are part of the implementation of the built-in admin UI and are generally not necessary if you just want to manipulate documents in a particular locale, with a pointer to how that is done via the piece and page REST APIs. Otherwise a person (or bot) could spend a lot of time obsessing with this page I suspect.

Reply:
Added.